README.md

```diff
@@ -111,6 +619,91 @@ const jsxFormatter: Formatter<JSX.Element> = {
 }
 ```
 
+</details>
+
+<details>
+<summary>Vue (using render functions)</summary>
+
+This formatter wraps the document in a `<div>`, each paragraph in a `<p>` and
+each text node in a `span` while applying a subset of possible markup properties.
+
+```ts
+ const f: Formatter<VNode> = {
+  default: (_e, children) => h('section',children()),
+  nodes: {
+    doc: (_node, children) => h('div', children()),
+    paragraph: (_node, children) => h('p', children()),
+    text: (node) => h('span',node.text)
+  },
+  marks: {
+    text: {
+      strong: (_mark, next) => h('b', next()),
+      underline: (_mark, next) => h('u', next()),
+      em: (_mark, next) => h('i', next()),
+      code: (_mark, next) => h('code', next()),
+      strike: (_mark, next) =>
+        h('span', { style: { textDecoration: 'line-through' } }, next()),      
+    },
+  },
+}
+   
+```
+
+</details>
+
+<details>
+<summary>Counting characters</summary>
+
+This example counts the characters of all text content. It's not the intended
+usage of formatters, but shows their flexibility nicely.
+
+Note the `default` callback: Without it, it would to nothing at it would never
```

Comment:
Reading the whole sentence now I feel this sounds a bit off. How about rephrasing it to something like this:

> Without the `default` callback, it would do nothing as it would never recurse into child nodes from any element including the `doc` root.

Just a thought :)